google-cloud-bigquery-storage/synth.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import synthtool as s
+import synthtool.gcp as gcp
+import synthtool.languages.ruby as ruby
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICMicrogenerator()
+library = gapic.ruby_library(
+    "bigquery/storage", "v1",
+    generator_version="0.2.0",
+    generator_args={
+        "ruby-cloud-gem-name": "google-cloud-bigquery-storage",
+        "ruby-cloud-title": "BigQuery Storage",
+        "ruby-cloud-summary": "The BigQuery Storage API provides fast access to BigQuery managed storage.",
```

Comment:
@quartzmo Can you change the key from "ruby-cloud-summary" to "ruby-cloud-description". We're making this change across all the microgenerator libraries to set reasonable long descriptions for rubygems (since we can already generate a good default for summary, e.g. "The official client library for the <> API"

Reply:
Done and regenerated.